treeherder/push_health/tests.py

```diff
@@ -146,13 +139,19 @@ def get_current_test_failures(push, option_map, jobs, investigatedTests=None):
                 'suggestedClassification': 'New Failure',
                 'confidence': 0,
                 'tier': job.tier,
+                'totalFailures': 0,
+                'totalJobs': 0,
                 'failedInParent': False,
-                'passFailRatio': passFailRatio,
```

Comment:
What consumers do we have of this data currently? Is it safe to remove it right away? It might be better to keep the old code in, and only remove it once the consumers have updated.

Reply:
this might make sense; my understanding is that only pushhealth consumes this, but with an API you never know.  I have added Sarah to the reviewer list and will see what else shakes out.

Reply:
I looked in papertrail and I didn't see any GET requests for the `health` and `health_summary` APIs that didn't originate from Treeherder so this is probably an ok change to make. We do only have 3 days of active data in the logs before its archived though.